Automattic/wp-calypso
# Framework: Replace lru-cache with lru

`lru-cache` has some old dependencies that don't play well with Webpack2.  `i18n-calypso` recently moved to `lru` which has the same interface with none of the baggage.

This patches teaches wp-calypso to use `lru` instead of `lru-cache`.

See https://github.com/Automattic/i18n-calypso/pull/18

To test, 
1. Load the Reader. Navigate from the following stream to a tag stream to a search stream to the following stream
2. Load themes and the theme browser. Everything should work.
